### PR TITLE
[doc] PR for PLAT-2196 and PLAT-2198 

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2196 and https://yugabyte.atlassian.net/browse/PLAT-2198

Preview: https://deploy-preview-10532--infallible-bardeen-164bc9.netlify.app/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check/

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ You can access Yugabyte Platform health monitor and configure alerts by navigati
 
 ![Configure alerts](/images/yp/config-alerts1.png)
 
-The **Alert Creation** view allows you to create new alerts for specific universes or for your instance of Yugabyte Platform, as well as configure or delete existing alerts.
+The **Alert Creation** view allows you to perform the following for specific universes or for your instance of Yugabyte Platform: 
+
+- Create new alerts.
+- Configure, delete, activate or deactivate existing alerts via **Actions**.
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:24]: activate, or deactivate

**Review by ddhodge — APPROVED** [review]
minor comments

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:24]: Fixed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md [hunk 2]
```diff
@@ -30,9 +33,11 @@ Regardless of the alert level, you create and configure an alert as follows:
...
 
-  Templates are available for alerts related to Yugabyte Platform operations, YugabyteDB operations, as well as YSQL and YCQL performance. 
+  Templates are available for alerts related to Yugabyte Platform operations, YugabyteDB operations, as well as YSQL and YCQL performance.<br>
+
+  The **Active** field allows you to define the alert as initially active or inactive.<br>
 
-  The **Threshold** field allows you to define the value (for example, number of milliseconds, resets, errors, nodes) that must be reached in order to trigger the alert.
+  The **Threshold** field allows you to define the value (for example, number of milliseconds, resets, errors, nodes) that must be reached in order to trigger the alert.<br>
 
   The **Destination** field allows you to select one of the previously defined recipients of the alert.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md:43]: Fixed

**Comment by lizayugabyte** [comment]
Merging this PR. (Alexander M provided input, which was reflected in the changes)

**Pull request merged**